lib/simp/cli/config/items/action/warn_client_yum_config_action.rb

```diff
@@ -10,26 +9,26 @@ def initialize(puppet_env_info = DEFAULT_PUPPET_ENV_INFO)
       @key             = 'yum::repositories::client::config::check'
       @description     = 'Check YUM configuration for SIMP clients'
       @category        = :sanity_check
-      @warning_message = <<DOC
-Unable to verify YUM configuration for SIMP clients.  Please
-manually verify prior to kickstarting SIMP clients.
+      @warning_message = <<~DOC
+        Unable to verify YUM configuration for SIMP clients.
```

Comment:
@lnemsick-simp  Does this need to be tabbed, or can it be spaces to match the rest of the code?

Reply:
The HEREDOC is indented appropriately with spaces, so I do not understand the question.

Reply:
Oh...I must have mis-read the diff, sorry!